### DISCUSSION TITLE
Focus on your own text editor.

### DISCUSSION THREAD

**Discussion by fedetango** [dailydotdev/daily#1230] [Feature Requests]
Congratulations! The platform seems fantastic to me.

It is comfortable, fluid, navigable UNTIL I decide to read a POST.

I was disappointed to see that most of the articles are sent to me outside the web.

DON'T BE AFRAID TO COMPETE WITH BLOGS.

They have all the potential to do it.

My suggestion is that ALL articles should be native to daily.dev.

Then, if they want, they add a button where the author can take the user to other sites where they have the same article.

When you click on a POST, the screen SHOULD NOT RELOAD.

Simply make a horizontal scrolling animation that clears the screen and brings up the item in question.

Leaving the article is as easy as continuing to browse in the same way as going from post to post on a social network.

And it is essential that the reading screen is as clean as possible with large spaces.
Get inspired by Medium.

And give importance to the text editor.
So far there is no blogging app that has a good cross-platform editor. Sometimes something as simple as dark mode is missing (Linkedin and Medium do not have it incredibly)

Do these things and I promise you that you will multiply the reach of daily.dev by 10

All the best!

**Comment by idoshamun** [comment]
Hi @fedetango,

Thanks for your comprehensive suggestion.
Since daily.dev aggregates content from other platforms we're always considering how to make it a win-win situation.
Showing the full content on daily.dev requires a lot of thinking from our side on how to keep it still a win-win to everyone.
Hence, we're not rushing into it but I agree that it can take the UX to the next level 🚀
